Core/Systems/Affixes/Affix.cs

```diff
@@ -6,15 +6,18 @@ namespace PathOfTerraria.Core.Systems.Affixes;
 
 internal abstract class Affix
 {
-	protected float MinValue;
-	protected float MaxValue = 1f;
+	public float MinValue;
+	public float MaxValue = 1f;
 
-	protected float Value = 1f;
+	public float Value = 1f;
 	// to a certain degree, none of the above is useable by the MobAffix...
 
 	public virtual void Roll()
 	{
-		Value = Main.rand.Next((int)(MinValue * 10), (int)(MaxValue * 10)) / 10f;
+		if (Value == 0)
```

Comment:
I think i have already written this somewhere; but it would probably be smarter to just set min and max value to the same number instead of setting the value; if we do it like this we wont be able to reroll any items.

*can fix in the future